cmp/tab/TabContainerModel.js

```diff
@@ -48,6 +51,8 @@ export class TabContainerModel {
      *      be route-enabled, with the route for each tab being "[route]/[tab.id]".
      * @param {string} [c.switcherPosition] - Position of the switcher docked within this component (or 'none').
      *      Valid values are 'top', 'bottom', 'left', 'right', 'none'.
+     * @param {boolean} [c.track] - Enables tracking of tab views in admin logger under category
```

Comment:
I would say "True to enable activity tracking of tab views" - that uses the same language as the admin client itself, vs "admin logger".